src/node/rpc/node_call_types.h

```diff
@@ -60,7 +60,7 @@ namespace ccf
       QuoteInfo quote_info;
       crypto::Pem public_encryption_key;
       CodeDigest code_digest;
-      std::optional<RawPolicy> security_policy;
+      std::optional<HostDataMetadata> security_policy;
```

Comment:
```suggestion
      std::optional<HostDataMetadata> host_data_metadata;
```

Reply:
I've reverted this, this is where we pass the raw security policy in through the env variable, so here it is still explicitly a security policy